matiasbenedetto/google-fonts-to-wordpress-collection
# Add Google Fonts capability option

This adds the Google Fonts [capability option](https://developers.google.com/fonts/docs/developer_api#optimization) and sets it to `WOFF2`. This means the font assets that are downloaded will be `WOFF2` instead of `TTF`.

This may be easier to review by looking at each commit, as the update to all the font files is making GH very slow 😅

- Most relevant change, adding the capability param: https://github.com/WordPress/google-fonts-to-wordpress-collection/pull/3/commits/29b2efa1228308af8382c8bab3529bafc9c36437
- Some code quality tweaks: https://github.com/WordPress/google-fonts-to-wordpress-collection/pull/3/commits/38149ff7a5c2ad1206b02e3ba4544994b31ec967
- The extremely large diff of all the font assets being updated and changed to WOFF2 files: https://github.com/WordPress/google-fonts-to-wordpress-collection/pull/3/commits/27ea8c725c938f448ac95bbde0f79b35a3108b78

Closes https://github.com/WordPress/google-fonts-to-wordpress-collection/issues/1.

## Testing Instructions
- Run `npm run files`
- The font assets that are downloaded should all be `WOFF2` files
